src/handler/commandHandler.ts

```diff
@@ -271,15 +306,15 @@ export const handleInteractionEvent = (
             client
         );
     }
-    return Promise.reject(new Error("Interaction is not a command"));
+    return messageComponentInteractionHandler(interaction as MessageComponentInteraction, client);
```

Comment:
Moooooooment mal. Check halt vorher ob die Interaktion wirklich ne MessageComponentInteraction is.
`if(interaction.isMessageComponent())` oder so